generators/client/templates/react/src/main/webapp/app/config/notification-middleware.ts

```diff
@@ -13,12 +14,16 @@ export default () => next => action => {
    */
   return next(action).then(response => {
     if (action.meta && action.meta.successMessage) {
-      // toast(action.meta.successMessage, { type: toast.TYPE.SUCCESS });
+      if (typeof action.meta.successMessage === 'string') {
+        toast.success(action.meta.successMessage);
+      } else {
+        toast.success(action.meta.successMessage[0] + response.action.payload.data.id);
+      }
     }
     return Promise.resolve(response);
   }).catch(error => {
     if (action.meta && action.meta.errorMessage) {
-      // toast(action.meta.errorMessage, { type: toast.TYPE.ERROR });
+      toast.error(action.meta.errorMessage);
```

Comment:
may be we should show the error message in the error caught here by default so
`toast.error(action.meta.errorMessage || error.message);` similar to how we handle it in Angular notification interceptor. And then in the reducers return the `meta.errorMessage` filed only if we have a specific message to overriride

Reply:
The logic for this file should mimic `generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_notification.interceptor.ts` and the error handling should also handle the logic here `/generators/client/templates/angular/src/main/webapp/app/shared/alert/_alert-error.component.ts`